extensions/script/python/PythonScriptEngine.cpp

```diff
@@ -33,7 +33,7 @@ Interpreter *getInterpreter() {
 }
 
 PythonScriptEngine::PythonScriptEngine() {
-  auto intepreter = getInterpreter();
+  getInterpreter();
```

Comment:
There doesn't seem to be any magic global initialization happening here, so I suggest removing the calls altogether.

Reply:
I am a bit unsure about this. Checking back in #500 this seems to be always part of the initialization, but I do not know how those members actually work in the context of the python interpeter. Is there a way to test this?

Reply:
You're right, the `py::scoped_interpreter` member of `Interpreter` is actually initializing global state. Nevermind.